.github/workflows/build_and_test.yml

```diff
@@ -1074,7 +1074,7 @@ jobs:
     needs: precondition
     if: fromJson(needs.precondition.outputs.required).k8s-integration-tests == 'true'
     name: Run Spark on Kubernetes Integration test
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
https://github.com/actions/runner-images/issues/10636
<img width="807" alt="image" src="https://github.com/user-attachments/assets/ea91dd71-55ec-47c6-93e4-0bdb93a57116">
Because the GitHub community is still in the process of `migration` (sometimes it's `ubuntu-22.04`, sometimes it's `ubuntu-22.04`), adding this is to verify the issue. After obtaining `approve`, I will remove it.